viewsgetter.bat

```diff
@@ -12,14 +12,15 @@ REM this gets all the data that has to do with the user's repos, but we dont rea
 curl "https://api.github.com/users/%user%/repos">%cd%\Repos.txt
 
 REM repo finder extracts the repo names so that we can do something with it
-javac -d out src/com/github/jojo2357/githubviewslogger/*.java
-java -cp $PWD/out/ com.github.jojo2357.githubviewslogger.RepoRefiner
+if not exist "out" mkdir out
+javac -d out src\com\github\jojo2357\githubviewslogger\*.java
+java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.RepoRefiner
 
 REM for each repo, we get taffic, and then parse that data
 for /f "delims=" %%x in (Repos.txt) do (
 curl "https://api.github.com/repos/%user%/%%x/traffic/views" -u %user%:%password%>%cd%\%%x.txt
-java com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Views
+java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Views
 curl "https://api.github.com/repos/%user%/%%x/traffic/clones" -u %user%:%password%>%cd%\%%x.txt
-java com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Clones
+java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Clones
```

Comment:
curl "https://api.github.com/repos/%user%/%%x/traffic/views" -u %user%:%password%>%cd%\%%x.txt
java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Views %directory%
curl "https://api.github.com/repos/%user%/%%x/traffic/clones" -u %user%:%password%>%cd%\%%x.txt
java -cp %cd%\out\ com.github.jojo2357.githubviewslogger.GitHubDataParser %%x Clones %directory%

Reply:
must be some rebasing artifacts, will add

Reply:
did you forget a \ cuz i think %cd%%%x should be %cd%\\%%x

Reply:
um it disappeared on me. copy paste strikes somehow?

Reply:
i think i actually updated the file tho it says outdated